torch_xla/csrc/status.h

```diff
@@ -143,6 +146,8 @@ absl::Status MaybeWithNewMessage(const absl::Status& status, const char* file,
                                  int32_t line,
```

Comment:
Update the comment of this function to say that the file location is appended to the payload instead of the error message?